### ISSUE TITLE
When loading singleplayer survival world, theres a crash

### ISSUE THREAD

**Issue by idkimanerd** [GTNewHorizons/GT-New-Horizons-Modpack#9790]
### Your GTNH Discord Username

Some Nerd With A Keyboard#4823

### Your Pack Version

2.1.2.2QF2

### Your Server

SP

### Type of Server

Single Player

### Your Actions

Loading survival world

### Crash Report

https://gist.github.com/idkimanerd/92224cbabc0c2b626b3e4ca3b0a2aa27

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the crash still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this crash consistently by following the exact steps I described above.
- [ ] I have asked other people and they confirm they also crash by following the exact steps I described above.

**Comment by bombcar** [comment]
did you by any chance join a multiplayer world first?

**Comment by idkimanerd** [comment]
No, but according to the discord it's a common issue with no known fix other than waiting longer on the title screen. That worked though.

**Comment by mitchej123** [comment]
It's a vanilla bug.  The workaround is playing on MP (locally hosted or otherwise).  There's some ASM fixes around that reduce the chance, but it's an annoying race condition that's hard to fix.
